pages/about/news/[slug].vue

```diff
@@ -0,0 +1,183 @@
+<!-- eslint-disable no-console -->
+
+<script setup>
+// HELPERS
+import _get from 'lodash/get'
+import format from 'date-fns/format'
+import removeTags from '../utils/removeTags'
+
+// GQL
+import ARTICLE_DETAIL from '../gql/queries/ArticleDetail.gql'
+
+const { $graphql } = useNuxtApp()
+
+const { data } = await useAsyncData('news-story', async () => {
+  const data = await $graphql.default.request(ARTICLE_DETAIL, { slug: params.slug })
+
+  if (!data.entry) {
+    error({ statusCode: 404, message: 'Page not found' })
+  }
+
+  // Elastic search?
+  // if (data) {
```

Comment:
@pghorpade Should this entire conditional be disabled for now? Given the ref ($elasticsearchplugin), it seems to depend on Elastic Search.

Reply:
Yes